ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/moreh_linear_backward_pybind.cpp

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_linear_backward_pybind.hpp"
+
+#include "pybind11/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_linear_backward/moreh_linear_backward.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear_backward {
+void bind_moreh_linear_backward_operation(py::module& module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_linear_backward,
+        "Moreh Linear Backward Operation",
+        ttnn::pybind_arguments_t{
+            py::arg("output_grad"),
+            py::arg("input"),
+            py::arg("weight"),
+            py::arg("are_required_outputs") = std::vector<bool>{true, true, true},
+
+            py::arg("bias") = std::nullopt,
+            py::arg("input_grad") = std::nullopt,
+            py::arg("weight_grad") = std::nullopt,
+            py::arg("bias_grad") = std::nullopt,
+
+            py::arg("input_grad_mem_config") = std::nullopt,
+            py::arg("weight_grad_mem_config") = std::nullopt,
```

Comment:
```suggestion
            py::arg("input_grad_memory_config") = std::nullopt,
            py::arg("weight_grad_memory_config") = std::nullopt,
```

Reply:
resolved in [9b40fca](https://github.com/tenstorrent/tt-metal/pull/12812/commits/9b40fca6fe2a092b7588eaeac2d24d998ecb517d)